Lombiq.Tests.UI.Shortcuts/Lombiq.Tests.UI.Shortcuts.csproj

```diff
@@ -34,23 +34,23 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Tenants" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Themes" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Users" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Media.Abstractions" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Recipes.Abstractions" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Abstractions" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.ResourceManagement.Abstractions" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Workflows" Version="1.6.0" />
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Tenants" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Themes" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Users" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Media.Abstractions" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Recipes.Abstractions" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Abstractions" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.ResourceManagement.Abstractions" Version="1.7.0" />
+    <PackageReference Include="OrchardCore.Workflows" Version="1.7.0" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(NuGetBuild)' != 'true'">
     <ProjectReference Include="..\..\..\src\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.OrchardCore\Lombiq.HelpfulLibraries.OrchardCore.csproj" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(NuGetBuild)' == 'true'">
-    <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="7.0.0" />
+    <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="7.0.1-alpha.8.osoe-682" />
```

Comment:
You were supposed do publish NuGet alphas after all PRs are approved, since if there's and review feedback, you'll need to do a release again. Hopefully, there won't be.

Reply:
I only did alpha releases for those projects that are being dependent on by other projects to see if there would be any issue.